stonesoup/simulator/simple.py

```diff
@@ -92,18 +92,58 @@ class MultiTargetGroundTruthSimulator(SingleTargetGroundTruthSimulator):
         default=0.1, doc="Probability of track dying in each time step. Default 0.1.")
     seed: Optional[int] = Property(default=None, doc="Seed for random number generation."
                                                      " Default None")
+    preexisting_states: Tuple = Property(
```

Comment:
I think the type for this should be `Collection[GaussianState]`

Reply:
I made a commit to change this. However I made it a Collection[StateVector] because they won't have covariances matrices. I also wanted to make it clear that you are supposed to pass in state vectors only. If it was a State, then it would require a timestamp. I think that could be confusing because it might lead people to think they can dictate when they want the target to appear in the ground truth. But what I have done here has all the states begin at time t=0.

Reply:
Ah, I misread the code and thought it was using covar to sample, but that was `initial_state` I was looking at. 👍